examples/http-response with function node to TLE

```diff
@@ -0,0 +1 @@
+[{"id":"736f681c.fb3ea8","type":"inject","z":"6eeba0c2.aa0738","name":"","topic":"","payload":"","payloadType":"date","repeat":"","crontab":"","once":false,"onceDelay":0.1,"x":120,"y":560,"wires":[["70029dca.401c34"]]},{"id":"70029dca.401c34","type":"http request","z":"6eeba0c2.aa0738","name":"","method":"GET","ret":"txt","paytoqs":false,"url":"https://celestrak.com/satcat/tle.php?CATNR=44109","tls":"","persist":false,"proxy":"","authType":"","x":130,"y":620,"wires":[["dcbe4635.b98e6"]]},{"id":"dcbe4635.b98e6","type":"function","z":"6eeba0c2.aa0738","name":"Move the data to the override values","func":"var p = msg.payload.split(\"\\n\");\nmsg.satid = p[0]\nmsg.tle1 =  p[1]\nmsg.tle2 =  p[2]\nmsg.payload = Date.now()\n\nreturn msg;","outputs":1,"noerr":0,"x":210,"y":680,"wires":[["4f94285d.8cc2f"]]},{"id":"4f94285d.8cc2f","type":"tle","z":"6eeba0c2.aa0738","satid":"BLUEWALKER 1  ","tle1":"","tle2":"2 44105  97.4795 168.9341 0059630 107.2691 253.5087 15.31277171 57050","coordsys":"latlongdeg","name":"(TLE prefilled with BLUEWALKER 1 data)","x":220,"y":740,"wires":[["f26379b6.1eb2f8"]]},{"id":"f26379b6.1eb2f8","type":"debug","z":"6eeba0c2.aa0738","name":"Message coming out of TLE node","active":true,"tosidebar":true,"console":false,"tostatus":false,"complete":"true","targetType":"full","x":200,"y":800,"wires":[]}]
```

Comment:
I think a better name could be along the lines of "set Satellite and TLE via msg"

Reply:
Ok by me

Reply:
I meant the name of the file  (as that is what is shown in the import menu).... ;-)

Reply:
Ahhhhh done